[Federation] Merging SDLs into base schema and creating engineV2 configuration from SDLs

chedom opened the pull request:
1. Merging SDLs into base schema
2. Creating engineV2 from SDLs (and base schema)
3. Adding example of creating and configuring federation gateway server

jensneuse commented:
@chedom this looks great! I've tried the example, works as expected. I'll review the code and merge it soon.

chedom commented:
> @chedom this looks great! I've tried the example, works as expected. I'll review the code and merge it soon.

Subscription operation with variables with graphql data source didn't work.
I have fixed it and added Subscription with variables to the example.
